vpr/src/place/net_cost_handler.cpp

```diff
@@ -88,14 +94,33 @@ static vtr::vector<ClusterNetId, NetUpdateState> bb_updated_before; // [0...clus
  * layer_ts_bb_coord_new are used.
```

Comment:
Also mention: layer_sink_pin_count is only used in the layer bounding box case.
Add @brief at the start of this.